src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -197,6 +197,7 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 
 		public override void Draw(Canvas? canvas)
 		{
+			ArgumentNullException.ThrowIfNull(canvas);
 			Controls.Compatibility.Platform.Android.CanvasExtensions.ClipShape(canvas, Context, Element);
 
 			base.Draw(canvas);
```

Comment:
This `Draw` method is in a public, shipped MAUI API. I found it might be easier to add a null check here that should not possibly happen?

We were getting a nullability warning calling `base.Draw()`.